engine/tox.ini

```diff
@@ -5,6 +5,7 @@ extend-exclude = */migrations/*
 ban-relative-imports = parents
 banned-modules =
   jinja2.Template = Use apply_jinja_template instead
+  django.apps.apps = Import models directly instead
```

Comment:
Adding a rule to ban `django.apps.apps`

Reply:
get_model is required in the migrations, would this rule allow that?

Reply:
yes, migration directories are ignored [here](https://github.com/grafana/oncall/pull/2619/files#diff-e1f096d06b7cdece59d58be9c27f1aee24c15c02c84b0063b35edcfb0d6fb528R4)